src/include/sof/audio/pcm_converter.h

```diff
@@ -20,6 +20,21 @@
 
 struct comp_buffer;
 
+#define CONFIG_PCM_CONVERTER_GENERIC
+
+#if __XCC__
+
+#include <xtensa/config/core-isa.h>
+
+#if XCHAL_HAVE_HIFI3
+
```

Comment:
Do, we want to force the usage of the optimized conversion if HIFI3 is set right?

Reply:
I would assume so. It's like that for all the component. In the future we could add option to config to toggle implementations.